### DISCUSSION TITLE
Difference between "Automation Behaviour" modes

### DISCUSSION THREAD

**Discussion by TheDoctor40** [darkreader/darkreader#9975] [Q&A] [answered]
I feel silly for not finding the answer anywhere, but I have one simple question which I can't figure out. I'm using the "new" design and using automation based on location. In the automation settings, there is a "Automation behaviour" setting, which you can toggle between "Toggle dark/light" and "Toggle on/off". What exactly is the difference between the two?

**Comment by Gusted** [comment, marked as answer]
Hi @TheDoctor40!

The automation behavior modes specifies what to do when the automation kicks in. So in the case of:
- "Toggle on/off": It will disable or enable Dark Reader accordingly (In one case Dark Reader just completely disabled). 
- "Toggle dark/light": It will set the theme to dark or light accordingly (Dark Reader still active in both cases).

**Reply by TheDoctor40** [reply]
I see, in hindsight that acually makes sense, don't know why I couldn't think of that haha. Thanks for the answer!
